misk/src/main/kotlin/misk/wire/ProtoMessageHttpClient.kt

```diff
@@ -0,0 +1,48 @@
+package misk.wire
+
+import com.squareup.moshi.Moshi
+import com.squareup.wire.Message
+import misk.web.mediatype.MediaTypes
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import javax.inject.Inject
+
+/**
+ * This helper class is for making HTTP requests to an endpoint that handles protobuf Message claases.
+ */
+class ProtoMessageHttpClient {
+  @Inject
```

Comment:
nit: inject this as a ``val`` in the constructor

Reply:
Sure, makes sense.